webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricStudentButton.js

```diff
@@ -13,7 +13,7 @@ export class SakaiRubricStudentButton extends rubricsApiMixin(RubricsElement) {
     evaluatedItemId: { attribute: "evaluated-item-id", type: String },
     evaluatedItemOwnerId: { attribute: "evaluated-item-owner-id", type: String },
     forcePreview: { attribute: "force-preview", type: Boolean },
-    instructor: { type: Boolean },
+    instructor: { type: String },
```

Comment:
I've used this pattern of 
`instructor: { type: Boolean }`
and have witnessed that it does work well.
I am just curious why here it does not?

Reply:
I did a few changes recently where I used 
`someVariable: { type: Boolean }`
and it worked fine

I think your right though that it should be investigated as to why it is not working here